include/ur.py

```diff
@@ -843,6 +843,7 @@ class ur_usm_alloc_info_v(IntEnum):
     BASE_PTR = 1                                    ## Memory allocation base pointer info
     SIZE = 2                                        ## Memory allocation size info
     DEVICE = 3                                      ## Memory allocation device info
+    POOL = 4                                        ## Memory allocation pool handle
```

Comment:
Did you mean "pool info" in the comment?

Reply:
Not really. Well, 'info' would be consistent with the other ones, but this option is actually used to obtain a 'handle' to the pool

Reply:
I didn't notice this discussion and [commented](https://github.com/oneapi-src/unified-runtime/pull/348#discussion_r1137327335) again on this. I think the *device info* is a mistake and should be *device handle* in the docs.